api/v1alpha1/volumereplicationgroup_types.go

```diff
@@ -18,53 +18,38 @@ package v1alpha1
 
 import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+
+	vrv1alpha1 "github.com/kube-storage/volume-replication-operator/api/v1alpha1"
```

Comment:
Let's change from the shim to the real implementation as a separate PR, this ewquires changes to the function `createVolumeReplicationCRForPVC` and also a level of readiness of kube-storage.

This is currently an unused/harmless import that can be removed.

Reply:
The ReplicationState definition comes from this import.  Only this file uses this import for now. The controller continues to use the shim and as you said, let's switch to the real VR implemention for the controller code changes in a separte PR.